### BIG differences, this is diff of wrong files.
### CLUS "Type Specifier OR"
### and CLHS topic selection, I don't find that page
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:or.txt	2017-04-27 21:39:47.000000000 +0100
+++ saved-pages-clus/cl:types:or.txt	2017-04-27 21:39:47.000000000 +0100
@@ -1,4 +1,34 @@
-OR
-Please select which reference to OR you intended
+Type Specifier OR

-MacroType Specifier
\ No newline at end of file
+Compound Type Specifier Kind
+Combining.
+Compound Type Specifier Syntax
+or typespec*
+Compound Type Specifier Arguments
+typespec - a type specifier.
+Compound Type Specifier Description
+This denotes the set of all objects of the type determined by the union of the typespecs. For example, the type list by definition is the same as (or null cons). Also, the value returned by position is an object of type (or null (integer 0 *)); i.e. either nil or a non-negative integer.
+* is not permitted as an argument.
+The type specifiers (or) and nil are equivalent. The symbol or is not valid as a type specifier; and, specifically, it is not an abbreviation for (or).
+\
\ No newline at end of file
